funding/manager.go

```diff
@@ -2549,14 +2560,32 @@ func makeFundingScript(channel *channeldb.OpenChannel) ([]byte, error) {
 // confirmation, and then to notify the other systems that must be notified
 // when a channel has become active for lightning transactions.
 // The wait can be canceled by closing the cancelChan. In case of success,
-// a *lnwire.ShortChannelID will be passed to confChan.
+// a *lnwire.ShortChannelID will be passed to confChan. If this function
+// detects that the inputs to the funding transaction have been spent by a
+// different transaction, an error will be sent along the spentChan. The
+// spentChan MUST be buffered.
 //
 // NOTE: This MUST be run as a goroutine.
 func (f *Manager) waitForFundingConfirmation(
 	completeChan *channeldb.OpenChannel, cancelChan <-chan struct{},
-	confChan chan<- *confirmedChannel) {
+	confChan chan<- *confirmedChannel, spentChan chan<- error) {
```

Comment:
`spentChan` can just be `errChan` here? 
Optionally pull the various channels out into a struct.